<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tmc::UI::QCachePasswordPage</class>
 <widget class="QWizardPage" name="tmc::UI::QCachePasswordPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::PasswordEchoOnEdit</enum>
     </property>
     <property name="placeholderText">
      <string>Choose a password for your cache</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::PasswordEchoOnEdit</enum>
     </property>
     <property name="placeholderText">
      <string>Confirm</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
